packages/core/src/components/button/_button.scss

```diff
@@ -260,6 +261,17 @@ a.pt-button {
   @extend .pt-button:disabled;
 }
 
+.pt-button-loading-spinner {
```

Comment:
Note that the `pt-button-loading-spinner` (perhaps poorly named) is not the `.pt-spinner`. It is the extra element discussed below